packages/react/src/themes/base/components/Checkbox/checkboxVariables.ts

```diff
@@ -0,0 +1,77 @@
+import { pxToRem } from '../../../../lib'
+
+export type CheckboxVariables = {
+  checkboxBackground: string
+  checkboxBorderColor: string
+  checkboxBorderStyle: string
+  checkboxBorderRadius: string
+  checkboxBorderWidth: string
+  checkboxColor: string
+  checkboxMargin: string
+  checkboxPadding: string
+
+  toggleBackground: string
+  toggleBorderColor: string
+  toggleBorderStyle: string
+  toggleBorderRadius: string
+  toggleBorderWidth: string
+  toggleColor: string
+  toggleMargin: string
+  togglePadding: string
+
+  checkedCheckboxBackground: string
+  checkedCheckboxBorderColor: string
+  checkedCheckboxColor: string
+  checkedToggleColor: string
+  checkedToggleBackground: string
+  checkedToggleBorderColor: string
+  checkedTogglePadding: string
+
+  disabledColor: string
+  disabledCheckboxBackground: string
+  disabledCheckboxBorderColor: string
+  disabledCheckboxColor: string
+  disabledToggleColor: string
+  disabledToggleBackground: string
+  disabledToggleBorderColor: string
+
+  padding: string
+}
+
+export default (siteVars: any): CheckboxVariables => ({
+  checkboxBackground: 'transparent',
+  checkboxBorderColor: siteVars.colors.grey[300],
```

Comment:
Let's try to use the color scheme here, as this is a fresh new component ;)

Reply:
At least in teams theme :))

Reply:
As we decided not to add the color scheme now and make the component red-lined, let's be sure to track this work